src/tests/balancer/mod.rs

```diff
@@ -12,6 +13,7 @@ pub fn config(solver_addr: &SocketAddr) -> tests::Config {
 node-url = 'http://localhost:8545'
 [dex]
 endpoint = 'http://{solver_addr}/sor'
+rpc-url = 'http://localhost:8545'
```

Comment:
The config file is split into 2 sections (1 with generic arguments and 1 for arguments specific for the particular dex solver). Instead of adding another `rpc-url` argument to the balancer specific section you could just use the `node-url` from the generic section for your feature.